controllers/apply-next/views/eligibility.njk

```diff
@@ -11,14 +11,14 @@
             {{ formHeader(
                 backUrl = backUrl,
                 title = 'Eligibility Checker',
-                prefix = prefix if eligibilityStatus !== 'eligible' 
+                prefix = prefix if eligibilityStatus !== 'eligible'
             )}}
- 
+
             {% if eligibilityStatus === 'eligible' %}
                 <div class="s-prose u-constrained-wide">
                     <h2>Great! You're eligible to apply to {{ formTitle }}</h2>
                     {{ eligibility.successMessage | safe }}
-                    <p class="class="form-actions"">
```

Comment:
Haha, what.

Reply:
And yet the browser didn't know what to do with this? What gives??